### PR TITLE
BAEL-4230 - Adding Introduction to Spring Data JDBC Project

### PR DESCRIPTION
**Pull request by rshtishi**
In the tutorial project there was no existing Spring Data JDBC module. I created a new module that demonstrate a simple example of Spring Data JDBC repository.

### CODE REVIEW

File: parent-boot-2/spring-data-jdbc/src/main/java/com/baeldung/Application.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.baeldung;
```
[the remark this answers is not in the document]
Comment (rshtishi) [reply parent-boot-2/spring-data-jdbc/src/main/java/com/baeldung/Application.java:1]: done

File: parent-boot-2/spring-data-jdbc/src/main/java/com/baeldung/Application.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    @Autowired
+    private DatabaseSeeder dbSeeder;
+
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+
+    @Override
+    public void run(String... arg0) throws Exception {
+        ;
```
[the remark this answers is not in the document]
Comment (rshtishi) [reply parent-boot-2/spring-data-jdbc/src/main/java/com/baeldung/Application.java:32]: done

File: parent-boot-2/spring-data-jdbc/pom.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
[the remark this answers is not in the document]
Comment (rshtishi) [reply parent-boot-2/spring-data-jdbc/pom.xml:1]: done

File: persistence-modules/spring-data-jdbc/src/main/java/com/baeldung/springdatajdbcintro/DatabaseSeeder.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+public class DatabaseSeeder {
+
+    private static final Logger LOGGER = (Logger) LoggerFactory.getLogger(DatabaseSeeder.class);
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public void insertData() {
+        LOGGER.info("> Inserting data...");
+        jdbcTemplate.execute("INSERT INTO Person(first_name,last_name) VALUES('Victor', 'Hygo')");
```
Comment (KevinGilmore) [inline persistence-modules/spring-data-jdbc/src/main/java/com/baeldung/springdatajdbcintro/DatabaseSeeder.java:20]:
```suggestion
        jdbcTemplate.execute("INSERT INTO Person(first_name,last_name) VALUES('Victor', 'Hugo')");
```

File: persistence-modules/spring-data-jdbc/pom.xml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jdbc</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<scope>runtime</scope>
+		</dependency>
+	</dependencies>
+
```
Comment (pivovarit) [inline persistence-modules/spring-data-jdbc/pom.xml:29]:
```suggestion
```

**Pull request merged**